libs/react-components/tedi/src/components/closing-button/closing-button.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import { Meta, StoryFn, StoryObj } from '@storybook/react';
+
+import ClosingButton, { ClosingButtonProps } from './closing-button';
+
+/**
+ * [Figma ↗](https://www.figma.com/file/jWiRIXhHRxwVdMSimKX2FF/TEDI-Design-System-(draft)?type=designnode-id=4514-63815&m=dev)<br/>
+ * [Zeroheight ↗](https://zeroheight.com/1ee8444b7/p/30df1b-closing-button)
+ */
+
+const meta: Meta<typeof ClosingButton> = {
+  component: ClosingButton,
+  title: 'Tedi-ready/Components/ClosingButton',
+  parameters: {
+    status: {
+      type: [{ name: 'internalComponent' }],
```

Comment:
STAR uses current CloseButton also, to achive closable Tag's.
Probaly we can use New StatusBadge and old Tags togheter in UI. (Because it's the same color and size, but just a note.)

Reply:
So you would like ClosingButton to be exported from library as well?

Reply:
For STAR it's not needed, but maybe somebody else also uses it? so it would be a breaking change.